### ISSUE TITLE
crash on server (1.12.2)

### ISSUE THREAD

**Issue by SDUBZ** [TechReborn/RebornStorage#68]
Forge:1.12.2-2559
Reborn storage: RebornStorage-1.12.2-3.0.1.26
Reborn core: RebornCore-1.12.2-3.5.1.166

crash report: 
[crash-2017-12-04_11.43.38-server.txt](https://github.com/TechReborn/RebornStorage/files/1528024/crash-2017-12-04_11.43.38-server.txt)

**Comment by modmuss50** [comment]
humm I thought I fixed this. I will have to take another look. thanks for the report.

**Comment by frozenbrain** [comment]
We've had the same issue on our server with Reborn Core 3.5.0.162 and Reborn Storage 3.0.0.23. At first players could no longer connect to the server. We've then removed all Reborn Storage blocks from the world and shut the server down. After that we could no longer start the server. We then tried to update to the latest version of our modpack (Direwolf20 1.12 1.3.0) with Reborn Core 3.5.1.166 and Reborn Storage 3.0.1.26.

We were able to start the server again after removing all Reborn Storage nodes from refinedstorage_nodes.dat using an NBT editor.
